cmd/internal/base/base.go

```diff
@@ -110,6 +114,58 @@ func DescribeCmd() *cobra.Command {
 	}
 }
 
+func DiffCmd() *cobra.Command {
+	var (
+		dsn string
+		drv string
+		dir string
```

Comment:
```suggestion
		dir, dsn string
```
drv is not needed (see previous comment),